packages/pluggableWidgets/switch-web/src/Switch.editorPreview.tsx

```diff
@@ -0,0 +1,29 @@
+import { createElement, ReactElement } from "react";
+import { parseStyle } from "@mendix/piw-utils-internal";
+
+import { SwitchPreviewProps } from "../typings/SwitchProps";
+import { Switch } from "./components/Switch";
+
+export function preview(props: SwitchPreviewProps): ReactElement {
+    // TODO: The widget generator is out of sync with Studio Pro design mode. Change PIW preview props typing (class -> className) generation to remove the ts-ignore below
```

Comment:
Inst it better to also fix the typing generator?

Reply:
This will be addressed by Page Editor [here](https://mendix.atlassian.net/browse/PAG-1317)